meshtastic/interfaces/ble/connection.py

```
@@ -69,6 +68,99 @@ def _is_device_not_found_error(err: Exception) -> bool:
+    ----------
+    target : object
+        Candidate object inspected for mock-instance type.
+
+    Returns
+    -------
+    bool
+        ``True`` when ``target`` is a ``Mock`` or ``NonCallableMock``.
+    """
+    try:
+        from unittest.mock import Mock, NonCallableMock
+    except Exception:  # pragma: no cover - stdlib import should always exist
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)